src/sentry/tasks/integrations/sync_assignee_outbound.py

```diff
@@ -28,13 +29,13 @@ def sync_assignee_outbound(external_issue_id: int, user_id: Optional[int], assig
     has_issue_sync = features.has("organizations:integrations-issue-sync", organization)
     if not has_issue_sync:
         return
+    integration = integration_service.get_integration(integration_id=external_issue.integration_id)
 
-    integration = Integration.objects.get(id=external_issue.integration_id)
     # Assume unassign if None.
     user = user_service.get_user(user_id) if user_id else None
-
-    installation = integration.get_installation(organization_id=external_issue.organization_id)
-
+    installation = integration_service.get_installation(
+        integration_id=integration.id, organization_id=external_issue.organization_id
+    )
```

Comment:
Should we have a service method that combines these queries? There are many workflows in integrations that will need the integration and the organization_integration & installation. Having a service method to get all the context in one pass would help here, and likely in more places.

Reply:
That's probably true.  I do like that idea.

Reply:
Put together the combination service call, but still keeping installation separate since it isn't a serialized RPC call, it returns a python object :+1: